deploy/docs/SideBySidePrometheus.md

```diff
@@ -1,9 +1,18 @@
 # How to install our Prometheus side by side with your existing Prometheus
 
-When installing our Helm Chart it is possible to have more than one Prometheus server running in the same cluster. However, do note that you cannot have more than one Prometheus Operator running in the same cluster. This document will take you through the steps to set up Sumo Logic collection when you have an existing Prometheus Operator you wish to keep intact.
+When installing our Helm Chart it is possible to have more than one Prometheus server running in the same cluster.
+
+However, do note that you must either limit the scope of the interaction of the Prometheus Operator
+(by setting option `kube-prometheus-stack.prometheusOperator.namespaces.additional`)
+or disable Prometheus Operator in Sumo Logic's Kubernetes collection (existing Prometheus Operator must have `namespace` in which Sumo Logic's Kubernetes collection is installed in the scope of its interaction, please pay attention to following flags set for Prometheus Operator: `namespaces`, `deny-namespaces` and `prometheus-instance-namespaces`,
```

Comment:
I would break this line, and not sure how it works in english, but would add `,` after `is installed` and before `in which`